src/sentry/templates/sentry/plugins/bases/issue/not_configured.html

```diff
@@ -19,8 +19,9 @@ <h3>{{ title }}</h3>
                 {% endfor %}
             </ul>
         {% else %}
-            {% url 'sentry-configure-project-plugin' organization_slug=project.organization.slug project_slug=project.slug slug=plugin.slug as link %}
-            <p>{% blocktrans %}You still need to <a href="{{ link }}">configure this plugin</a> before you can use it.{% endblocktrans %}</p>
+            <p>{% blocktrans %}You still need to<a href="{% absolute_uri '/{}/{}/settings/plugins/' project.organization.slug project.slug p.slug %}">configure this plugin</a>
```

Comment:
This needs to be `/{}/{}/settings/plugins/{}/` the example you pulled from is wrong: https://github.com/getsentry/sentry/commit/74674b3a23a1c499785dbe670123573690260c59#diff-e5dd1cbab912859a222628ff9f30766f

This one needs an additional `{}/` at the end as well.